pkg/collect/collector_test.go

```diff
@@ -364,3 +365,83 @@ pwd=somethinggoeshere;`,
 		})
 	}
 }
+
+func TestCollector_DedupCollectors(t *testing.T) {
+	tests := []struct {
+		name       string
+		Collectors []*troubleshootv1beta2.Collect
+		want       []*troubleshootv1beta2.Collect
+	}{
+		{
+			name: "multiple cluster info",
+			Collectors: []*troubleshootv1beta2.Collect{
+				{
+					ClusterInfo: &troubleshootv1beta2.ClusterInfo{},
+				},
+				{
+					ClusterInfo: &troubleshootv1beta2.ClusterInfo{},
+				},
+			},
+			want: []*troubleshootv1beta2.Collect{
+				{
+					ClusterInfo: &troubleshootv1beta2.ClusterInfo{},
+				},
+			},
+		},
+		{
+			name: "multiple cluster resources with matching namespace lists",
```

Comment:
Just a comment. No change required.

If you reverse the list of namespaces to `[]string{"namespace2", "namespace1"}` then the specs are different, but I guess that's OK. Its better having some duplicates compared to removing a collector that shouldn't have been removed.

Reply:
right, yeah eventually we could do sorting of fields to garuantee this, but not going to address that with this PR